test/runtime/connectivity.go

```diff
@@ -280,135 +280,200 @@ var _ = Describe("RuntimeValidatedConntrackTest", func() {
+		}
 	})
-	It("Conntrack disabled", func() {
-		endpoints, err := vm.GetEndpointsIds()
-		Expect(err).Should(BeNil(), "could not get endpoints IDs")
 
-		status := vm.EndpointSetConfig(endpoints[helpers.Server], helpers.OptionConntrack, helpers.OptionDisabled)
-		Expect(status).Should(BeTrue(), fmt.Sprintf("could not set %s=%s on endpoint %q", helpers.OptionConntrack, helpers.OptionDisabled, helpers.Server))
-
-		status = vm.EndpointSetConfig(endpoints[helpers.Client], helpers.OptionConntrack, helpers.OptionDisabled)
-		Expect(status).Should(BeTrue(), fmt.Sprintf("could not set %s=%s on endpoint %q", helpers.OptionConntrack, helpers.OptionDisabled, helpers.Client))
+	It("Conntrack-related configuration options for endpoints", func() {
 
-		clientServerConnectivity()
-	})
-
-	It("ConntrackLocal disabled", func() {
+		By("ConntrackLocal configuration option disabled for each endpoint")
 		endpoints, err := vm.GetEndpointsIds()
 		Expect(err).Should(BeNil(), "could not get endpoint IDs")
 
-		status := vm.EndpointSetConfig(endpoints[helpers.Server], helpers.OptionConntrackLocal, helpers.OptionDisabled)
-		Expect(status).Should(BeTrue(), fmt.Sprintf("could not set %s=%s on endpoint %q", helpers.OptionConntrackLocal, helpers.OptionDisabled, helpers.Server))
+		conntrackOptionModes := []string{helpers.OptionDisabled, helpers.OptionEnabled}
+		for _, conntrackOptionMode := range conntrackOptionModes {
+			By(fmt.Sprintf("Testing ConntrackLocal endpoint configuration option: %s", conntrackOptionMode))
+			status := vm.EndpointSetConfig(endpoints[helpers.Server], helpers.OptionConntrackLocal, conntrackOptionMode)
+			Expect(status).Should(BeTrue(), fmt.Sprintf("could not set %s=%s on endpoint %q", helpers.OptionConntrackLocal, helpers.OptionDisabled, helpers.Server))
 
-		status = vm.EndpointSetConfig(endpoints[helpers.Client], helpers.OptionConntrackLocal, helpers.OptionDisabled)
-		Expect(status).Should(BeTrue(), fmt.Sprintf("could not set %s=%s on endpoint %q", helpers.OptionConntrackLocal, helpers.OptionDisabled, helpers.Client))
+			status = vm.EndpointSetConfig(endpoints[helpers.Client], helpers.OptionConntrackLocal, conntrackOptionMode)
+			Expect(status).Should(BeTrue(), fmt.Sprintf("could not set %s=%s on endpoint %q", helpers.OptionConntrackLocal, helpers.OptionDisabled, helpers.Client))
 
-		clientServerConnectivity()
-	})
+			clientServerConnectivity(true)
+		}
 
-	It("ConntrackLocal Enabled", func() {
-		endpoints, err := vm.GetEndpointsIds()
-		Expect(err).Should(BeNil(), "could not get endpoint IDs")
+		By("Testing Conntrack endpoint configuration option disabled")
 
-		status := vm.EndpointSetConfig(endpoints[helpers.Server], helpers.OptionConntrackLocal, helpers.OptionEnabled)
-		Expect(status).Should(BeTrue(), fmt.Sprintf("could not set %s=%s on endpoint %q", helpers.OptionConntrackLocal, helpers.OptionEnabled, helpers.Server))
+		status := vm.EndpointSetConfig(endpoints[helpers.Server], helpers.OptionConntrack, helpers.OptionDisabled)
```

Comment:
Same here

Reply:
Added a function that sets an option and then checks it for the provided endpoint for all cases in this test.